### PR TITLE
Appdev 9895 updating homestead

### PR DESCRIPTION
**Pull request by sharonluong**
This PR updates the base Homestead box from 6.4.0 to 10.1.0. This was necessary to update a few things, including node version. 6.4.0 was necessary when Jitterbug was still on PHP 5.4 but that is no longer the case.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -157,6 +157,43 @@ $ vagrant ssh
...
+    ```bash
+    if File.exist? customizationScriptPath then
+      config.vm.provision "shell", path: customizationScriptPath, privileged: false, keep_color: true
+    end
+    
+    if File.exist? solrRestartScriptPath then
+      config.vm.provision "shell", path: solrRestartScriptPath, run: 'always', privileged: false, keep_color: true
+    end
+    ```
+6. Archive your known_hosts file or delete the entry for `192.168.10.10`
```
Comment (cazzerson) [inline README.md:186]: Let's just recommend deleting the line, not the whole file.

File: README.md [hunk 2]
```diff
@@ -157,6 +157,43 @@ $ vagrant ssh
 $ cd /vagrant
 $ gulp
 ```
+
+## Updating Homestead
+1. Update the version specified in `Homestead.yaml.example`
```
Comment (cazzerson) [inline README.md:162]: "base box version"

**Review by cazzerson — CHANGES_REQUESTED** [review]
Looks great, thanks!

**Comment by sharonluong** [comment]
@cazzerson ready for re-review

**Pull request merged**